June 11th Launch (#10255)
getsentry/sentry-docs: docs/platforms/apple/common/tracing/index.mdx

---
title: Set Up Tracing

> description: "Learn how to enable tracing in your app and discover valuable performance insights of your application."

Reviewer: ```suggestion
description: "Learn how to enable tracing in your app and get valuable performance insights about your application."
```
